docusaurus/docs/running-tests.md

```diff
@@ -340,6 +340,32 @@ script:
 
 Follow [this article](https://medium.com/@knowbody/circleci-and-zeits-now-sh-c9b7eebcd3c1) to set up CircleCI with a Create React App project.
 
+### AWS CodeBuild
+
+1. Add a buildspec.yml file to your git repository.
+
+```yaml
+version: 0.2
+env:
+  variables:
+    CI: "true"
+phases:
+  install:
+    commands:
+      - echo Entered the install phase...
```

Comment:
It probably isn't necessary to print this comment.

Reply:
done